.github/workflows/release.yml

```diff
@@ -0,0 +1,95 @@
+name: trino-python-client release
+
+on:
+  workflow_dispatch:
+
+env:
+  PYTHON_VERSION: "3.9.5"
+
+jobs:
+  bump-version:
+    name: Bump trino-python-client version
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{env.PYTHON_VERSION}}
+
+      - name: Configure Git user
+        run: |
+          git config user.email "releases@trinodb.io"
+          git config user.name "Trino release automation"
+
+      - name: Set release version
+        run: |
+          echo "VERSION_NUMBER=$(./setup.py --version)" >> $GITHUB_ENV
+          echo "NEXT_VERSION_NUMBER=$(echo ${{ env.VERSION_NUMBER }} | awk -F. -v OFS=. '{$2 += 1 ; print}')" >> $GITHUB_ENV
```

Comment:
If changing version numbering scheme to something like `317` instead of `0.317.0` helps with this we can do that as well probably. It seems Trino moved from 0.x numbers to plain x but the Python client never followed.

cc: @martint @findepi

Reply:
Note that Python client never used Trino version numbering, only something similar.
I vaguely remember PyPI could be requiring "semver"-like, 3-part versions.
I may be wrong though, and also things could have changed.
From "product" perspective, I am fine with versioning Python client  with just one number (the major version).

Or, we can append ".0.0", if that's needed.
We had "0." at the front, because that's what Trino was using (back when it was Presto)

Reply:
Based on https://peps.python.org/pep-0440/#final-releases it looks like only 2 and 3 parts semver version is supported. As an alternative, trino-python-client keeps existing versioning and switch to semantic versioning and release 1.0.0 once we agree that a client is well-supported, mature and feature rich.

Reply:
It supports one or more parts - single part versions also work.

Reply:
So what are the next steps here? I don't really like singe part version number, it doesn't follow best practises in Python packages versioning.